src/Joomlatools/Console/Joomla/Util.php

```diff
@@ -190,4 +190,13 @@ public static function isJoomlatoolsBox()
 
         return false;
     }
+
+	/**
+	 * Return a portable path for Joomla
+	 *
+	 * @return string
+	 */
+    public static function getPath() {
+    	return sys_get_temp_dir() . '/.joomla';
```

Comment:
@msaladna How do you suggest we deal with the [template files](https://github.com/apisnetworks/joomlatools-console/tree/6ef1baadd26f24863539c7791e013a2c49b030c3/bin/.files) if we default to the system temporary path? 

A clean install would then not have access to these files since we're looking in another directory. We could make the distinction here between Phar and a normal script here, and keep returning the hardcoded path to the ./bin/.joomla directory for now. Only revert to `sys_get_temp_dir()` if inside a Phar. 

How would you see this work in your use case?

Reply:
Rename getPath to `getWriteablePath()`. Add second utility function, `getTemplatePath()`? Let me finish up these changes, but them through testing before commit. Also implemented #69